Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -16,4 +16,39 @@ extension Snapshotting where Value == URLRequest, Format == String {
 
     return ([method] + headers + body).joined(separator: "\n")
   }
+  
+  /// A snapshot strategy for comparing requests based on a cURL representation.
+  public static let curlRepresentation = SimplySnapshotting.lines.pullback { (request: URLRequest) in
```

Comment:
Let’s call this `curl`. Nice and simple 😄

Reply:
Sure thing! Done